.spyproject/codestyle.ini

```diff
@@ -0,0 +1,6 @@
+[codestyle]
+indentation = True
+
+[main]
+version = '0.1.0'
+
```

Comment:
I'm not sure where those came from.  I think they got swept up in the `git add -A`